### DISCUSSION TITLE
Crash and hot RTL (Windows)

### DISCUSSION THREAD

**Discussion by gm8arv** [merbanan/rtl_433#2113] [General]
Running on a slightly older, slower PC, Windows-10/64, a run once a minute occasionally stops with the following message.  No 433 MHz data is captured, so my job stops.

Registered 1 out of 134 device decoding protocols [ 3 ]
Found Rafael Micro R820T tuner
r82xx_write: i2c wr failed=-9 reg=0c len=7
[R82XX] PLL not locked!
Exact sample rate is: 250000.000414 Hz
[R82XX] PLL not locked!
Sample rate set to 250000 S/s.
Tuner gain set to Auto.

This has happened on two dongles (including nooelec), and the dongles were hot afterwards (normally just very slightly warm to the touch).  This is the log from the most recent run - the previous was similar but may not have been the same registers.  This is the script I run:

:start
echo %date% %time%
title %time% RTL_433 started...
rtl_433.exe -R 3 -E 1 > _temp.txt
timeout /T 2
RTL433toMRTG > rtl-humid-temp.txt
type _temp.txt
timeout /T 59
goto :start

The RTL433toMRTG is my own program converting the output from rtl_433 to MRTG format.

Perhaps you can see why it's failing?  Thanks.

**Comment by zuckschwerdt** [comment]
Can't really help with RTL-SDR or USB failures. But that program version is ancient. Try a current one from [Releases](https://github.com/merbanan/rtl_433/releases).
